plugins/org.eclipse.n4js/src/org/eclipse/n4js/scoping/members/MemberScopingHelper.xtend

```diff
@@ -113,6 +113,8 @@ class MemberScopingHelper {
 	 * Creates member scope via #members and decorates it via #decorate.
 	 */
 	private def IScope decoratedMemberScopeFor(TypeRef typeRef, MemberScopeRequest memberScopeRequest) {
+		if(typeRef === null)
+			return IScope.NULLSCOPE
```

Comment:
maybe add comment "broken AST" (to highlight why this fail-safe behavior was added here)